packages/amplify_datastore/test/amplify_datastore_delete_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:amplify_datastore/amplify_datastore.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import './testData/Post.dart';
+
+void main() {
+  const MethodChannel dataStoreChannel =
+      MethodChannel('com.amazonaws.amplify/datastore');
+
+  AmplifyDataStore dataStore = AmplifyDataStore(modelSchemas: null);
+
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  setUp(() {});
+
+  tearDown(() {
+    dataStoreChannel.setMockMethodCallHandler(null);
+  });
+
+  dynamic getJsonFromFile(String path) async {
+    path = 'resources/delete_api/' + path;
+    String jsonString = '';
+    try {
+      jsonString = await File(path).readAsString();
+    } catch (e) {
+      jsonString = await File('test/' + path).readAsString();
+    }
+    return jsonDecode(jsonString);
+  }
```

Comment:
We should probably pull this out similar to android.

Reply:
Done.